Source/Request/Request.JSONP.js

```diff
@@ -119,19 +119,20 @@ Request.JSONP = new Class({
 		return !!this.running;
 	},
 
-	clear: function(){
+	clear: function(index){
```

Comment:
Good stuff. Might suggest we used data-jsonp-index or something rather than the element attribute, but whatever, it doesn't matter really.